packages/send/frontend/src/stores/auth-store.ts

```diff
@@ -202,6 +224,24 @@ export const useAuthStore = defineStore('auth', () => {
     }
   }
 
+  async function loadUser() {
+    const user = await userManager.getUser();
+    if (user) {
+      // If user already loaded, exit.
+      return;
+    }
+    try {
+      const result = await browser.storage.local.get(STORAGE_KEY_AUTH);
+      if (result[STORAGE_KEY_AUTH]) {
+        const userInstance = new User(result[STORAGE_KEY_AUTH]);
+        await userManager.storeUser(userInstance);
+      }
+    } catch (e) {
+      console.log(`No error. Only works if running in add-on.`);
+      console.log(e);
+    }
+  }
+
```

Comment:
If there's not already a user loaded, we instantiate one using the auth cookie in storage.

We wrap the `browser.storage.local.get()` in a `try/catch` because this API is not present in the browser.